.gitpod.yml

```diff
@@ -1,8 +1,10 @@
 ---
 # List the start up tasks. Learn more https://www.gitpod.io/docs/config-start-tasks/
 tasks:
-  - name: Install npm depdencies
-    init: npm i
+  - name: Install npm dependencies
+    init: |
+      npm ci
+      npx playwright install --with-deps chromium
```

Comment:
It didn't work for me, I still needed to run this command manually to make the test pass 🤔 

I wonder why this command doesn't run when building the project on GitHub